httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/CacheUpdateHandler.java

```diff
@@ -65,6 +65,25 @@
     /**
      * Creates a cache entry for the given request, origin response message and response content.
      */
+    public HttpCacheEntry createCacheEntry(
```

Comment:
@arturobernalg `CacheUpdateHandler` is not a part of public API. Deprecation is not needed  here.